### PR TITLE
Support importing Swarm files

### PR DESCRIPTION
**Pull request by axic**
Implements #356 / #357. Depends on #366.

This requires CORS enabled on swarm-gateways.net, which is in progress (https://github.com/ethereum/go-ethereum/issues/3350).

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -429,13 +430,23 @@ var run = function () {
...
+      handleGithubCall(match[3], match[4], function (err, content) {
+        if (err) {
+          cb('Unable to import "' + url + '": ' + err)
+          return
+        }
+
+        cachedRemoteFiles[url] = content
+        cb(null, content)
+      })
+    } else if ((match = /^bzzr:\/\/?([0-9a-fA-F]{64}$)/.exec(url))) {
```
Comment (chriseth) [inline src/app.js]: I think we should also support `bzz://` because that supports directory structures.

Comment (axic) [inline src/app.js]: The rule for that would be something like `^(bzz:\/\/?[0-9a-fA-F]{64}/.*)$`. Probably better to catch both with `bzzr?` and let the Swarm client do its work.

**Comment by axic** [comment]
Need to update `swarmgw` first to support other than `bzzr://<hash>` before merging this.

**Comment by axic** [comment]
Added a question to geth: https://github.com/ethereum/go-ethereum/issues/3523

**Comment by axic** [comment]
There's still no CORS on the endpoint...

**Comment by axic** [comment]
Supports all three Swarm URLs and CORS is now enabled on the gateways. Can be merged.

File: src/app.js [hunk 2]
```diff
@@ -515,12 +516,36 @@ var run = function () {
       })
   }
 
+  function handleSwarmImport (url, cb) {
+    swarmgw.get(url, function (err, content) {
+      // retry if this failed and we're connected via RPC
+      if (err && !executionContext.isVM()) {
```
Comment (chriseth) [inline src/app.js]: At some point, we should reverse the order here, I tihnk.

Comment (axic) [inline src/app.js]: Unfortunately we have to remove the RPC support because it will be removed, I have been told by the Swarm team.

**Pull request merged**